schemas/document-generation-started.go

```diff
@@ -0,0 +1,7 @@
+package schemas
+
+//  DocumentGenerationStarted struct for document-generation-started topic
```

Comment:
Please use only a single space after the double slash.

Reply:
Remove in [ce005fb](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/ce005fb8926cbc2899d61cf66de233a4e80ce97c)